### DISCUSSION TITLE
Support multiple repo connections for a GitBook project

### DISCUSSION THREAD

**Discussion by Wizzel1** [GitbookIO/community#49] [Feature Requests]
### What's your feature/idea?

It would be nice to be able to synchronize multiple repositories with a Gitbook project. 

### What's your desired outcome?

More flexibility for different repo structures.

### What's the impact of this for you?

- [X] Critical
- [ ] High
- [ ] Medium
- [ ] Low

**Comment by SamyPesse** [comment]
@Wizzel1 Curious to make sure I understand correctly: you are looking to connect multiple repositories to one GitBook space?

What is the use-case?

We'll probably not do it as it causes nasty side-effects, for example when reorganizing the pages and moving from one to another.

**Reply by Wizzel1** [reply]
We have 2 repos for our project (backend and frontend) and we would like to be able to edit the markdown files within the scope of a PR so it becomes clear to the reviewer if changes have been documented.

**Reply by ahcsco** [reply]
I agree. Should be supported.

**Reply by idlework** [reply]
I am also looking for a solution with Gitbook for this issue. It is possible with MkDocs and [multirepo plugin](https://github.com/jdoiro3/mkdocs-multirepo-plugin). But would prefer to stay with Gitbook.

**Reply by jessicarose** [reply]
I'm in the research phase for tooling and this would also be a feature I would love, as I've got a project with different types of docs in different repos and have a strong desire to not use MkDocs ever again if I can help it

**Reply by ploetzly** [reply]
I am doing documentation for a blockchain and they have multiple repos for different products and functions that live on the blockchain so we also need the ability to sync multiple repos and have them all still a part of the same Gitbook layout/not open a new tab for each one.
